mitosheet/mitosheet/streamlit/v1/apps/column_definitions_app.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import streamlit as st
+from mitosheet.streamlit.v1 import spreadsheet
+
+st.set_page_config(layout="wide")
+st.title('Tesla Stock Volume Analysis')
+
+df = pd.DataFrame({
+    'A': [1, 2, 3, 4, 5, 6, 7, 8, 9],
+    'B': [1, 2, 3, 4, 5, 6, 7, 8, 9]
+})
+new_dfs, code = spreadsheet(
+    df, 
+    column_definitions=[
+        [
+            {
+                'columns': ['A', 'B'],
+                'conditional_formats': [{
+                    'filters': [{'condition': 'greater_than_or_equal', 'value': 5}], 
+                    'font_color': '#c30010', 
+                    'background_color': '#ffcbd1' 
+                }] 
+            },
+            {
+                'columns': ['A'],
+                'conditional_formats': [{
+                    'filters': [{'condition': 'less', 'value': 2}], 
+                    'font_color': '#f30010', 
+                    'background_color': '#ddcbd1' 
+                }] 
+            }
+        ],
+        [
+            # Add code here to style the second dataframe
+        ]
```

Comment:
This errors, because you have a second dataframe list in here... so one can't run this app.